src/main/webapp/assets/js/place.js

```diff
@@ -26,7 +26,10 @@ $(document).ready( function () {
 	    'image_link': $('input[id = id-img]').val(),
 	    'tags': tags.join(":")
 	};
-	$.ajax({
+        if (formData.image_link === "") {
```

Comment:
are there only two cases? empty string and image path?
are we sure it will never be null?